WordPress/Classes/ViewRelated/Comments/Content/CommentDetailContentTableViewCell.swift

```diff
@@ -67,6 +67,7 @@ private extension CommentDetailContentTableViewCell {
             hostingController.rootView = content
         } else {
             let hostingController = UIHostingController<CommentContentHeaderView>(rootView: content)
+            hostingController.view.setContentCompressionResistancePriority(.required, for: .vertical)
```

Comment:
This change is not part of the fix, but I thought of including in this PR because it's just a 1 line change.

I noticed that sometimes the comment header is compressed ( it's height is 0 ) and a constraint warning is thrown in the console. Setting the compression resistance fixed that issue.